src/google-maps/google-map/google-map.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+  OnInit,
+  ViewChild,
+} from '@angular/core';
+import {ReplaySubject} from 'rxjs';
+import {take} from 'rxjs/operators';
+
+/**
+ * Angular component that renders a Google Map via the Google Maps JavaScript
+ * API.
+ * @see https://developers.google.com/maps/documentation/javascript/reference/
+ */
+@Component({
+  selector: 'google-map',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: '<div #map></div>',
+})
+export class GoogleMap implements OnInit {
+  @Input() height = '500px';
```

Comment:
Any particular reason to use these default values? If not, I'd add a comment like "Arbitrarily chosen default size", otherwise a comment with the rationale

Reply:
Yes, arbitrary. I will add the comment.